src/components/Selector.jsx

```diff
@@ -44,14 +44,13 @@ export default function Selector({templateInfo, animationManager, blinkManager,
     mousePosition,
     removeOption,
     saveUserSelection,
-    isLoading,
-    setIsLoading,
   } = useContext(SceneContext)
   const {
     playSound
   } = useContext(SoundContext)
   const { isMute } = useContext(AudioContext)
-  const {setLoading} = useContext(ViewContext)
+  const {loading, setLoading} = useContext(ViewContext)
```

Comment:
I prefer your naming convention of isLoading, isEffecting. These are proper names for booleans. Let's rename `loading` to `isLoading`. Effecting isn't a word so maybe `isPlayingEffect`? or something like that. :)